src/pages/lessons/[slug].tsx

```diff
@@ -186,7 +117,7 @@ const lessonLoader = (slug: any, token: any) => (query: string) => {
 }
 
 type LessonProps = {
-  initialLesson: LessonResource
+  initialLesson: Types.LessonResource
 }
```

Comment:
due to suggestion from the comment above about types import and naming convention

Reply:
Got it, will turn it back